File: libalgo/src/main/scala/com/beaconrep/libalgo/apps/WODecisionTree.scala
package com.beaconrep.libalgo.apps

import com.beaconrep.libalgo.core._
import com.beaconrep.libalgo._

//import org.apache.kafka.clients.producer.{KafkaProducer, ProducerConfig, ProducerRecord}

object WODecisionTree extends App {
  
  val frameWork = new SparkFramework("DecisionTree", "local[*]")
  
  val data = new facts(frameWork.getSession())
 
  data.initCSVDataPoint("resources/ParameterMaterialNoEmpty.csv")
  
  val algorithm = new SparkDecisionTree(frameWork.getSession(), data)
           
  println(algorithm.buildModel.scoreAndLabels);
  
}

File: libalgo/src/main/scala/com/beaconrep/libalgo/javaapp/WODecisionTree.java
package com.beaconrep.libalgo.javaapp;

import com.beaconrep.libalgo.algorithm;
import com.beaconrep.libalgo.core.*;

public class WODecisionTree {

  public static void main(String[] args) {
    SparkFramework frameWork = new SparkFramework("RandomForest", "local[*]");    
    algorithm our_algorithm = new SparkRandomForest(frameWork.getSession());
    our_algorithm.initCSVDataPoint("resources/ParameterMaterialNoEmpty.csv");
    //System.out.println(our_algorithm.buildModel().
  }
}
